3 zarr and re-chunking.ipynb

```diff
@@ -4,24 +4,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As seein in *'2. introduction to zarr'* notebook, zarr file have encodings called 'chunk'.  \n",
+    "# more zarr, rechunking\n",
+    "\n",
+    "As seein in [2. introduction to zarr](https://github.com/tinaok/Pangeo-for-beginners/blob/master/2%20Introduction%20to%20zarr.ipynb) notebook, zarr archives are made of 'chunks' which corresponds to files.\n",
     " \n",
     "How your data is 'chunk'ed plays important role on your computation time, through your cpu-time, and io-wait time.\n",
+    "For example:\n",
+    "\n",
+    "1. If chunk sizes are too small, calculations will create too many meta-data access and will slow down IO access. You may also slow down IO access of your colleagues.\n",
+    "2. If chunk order/size is not suitable to your computation, each dask workers need to exchange data by communication, thus your computational time gets slow.  \n",
```

Comment:
you -> we?

Reply:
Il faut changer les you en we un peu partout, je te laisse faire ça.